packages/docs/PropTables/SelectPropTable.tsx

```diff
@@ -114,7 +119,7 @@ const selectProps: Prop[] = [
 
   {
     name: 'onItemChange',
-    types: '(value: string | number | Array<string | number>) => void',
+    types: '(value: any | Array<any>, option: Option | Array<Option>) => void',
```

Comment:
`SelectOptionPropTable` needs the `title` changed to `Select[Item]` I believe.

(Left the comment here, since it would only let me add comments in changed code sections.)

Reply:
`Select` still uses `options` and `action` as items. Maybe I should change the prop `onItemChange` to `onOptionChange`?

Reply:
I know we've had some offline conversations about `items` vs. `options`. Maybe we need to have another one so that these are consistent?